app/views/series/show.html.erb

```diff
@@ -9,7 +9,7 @@
 </div>
 
 <div class="row mb-5">
-  <div class="col<%= "-9" if @series.image.attached? %>">
+  <div class="col<%= '-9' if @series.image.attached? %>">
```

Comment:
@cdoyle-temple I only see a closing double quote on this line.  There isn't an opening double quote.

Reply:
@sensei100 class="col ... %>">
                                ^               ^